.gitignore

```diff
@@ -2,4 +2,6 @@
 bin
 obj
 *.suo
-*.user
\ No newline at end of file
+*.user
+.vs/
+WiX.4.0.0.5512-pre/
```

Comment:
What's WiX.4.0.0.5512-pre? If it's supposed to be a NuGet package, I don't see any references.

Reply:
It's the nuget artifacts for wix installer nuget. I'd rather it here, as the references will be added.